akka-actor-tests/src/test/java/akka/pattern/StatusReplyTest.java

```diff
@@ -30,53 +30,41 @@
       new AkkaJUnitActorSystemResource("JavaAPI", AkkaSpec.testConf());
 
   @Test
-  public void testSuccessApi() {
+  public void successReplyThrowsExceptionWhenGetErrorIsCalled() {
     StatusReply<String> reply = StatusReply.success("woho");
     assertTrue(reply.isSuccess());
     assertFalse(reply.isError());
     assertEquals("woho", reply.getValue());
-    try {
-      reply.getError();
-      Assert.fail("Calling get error on success did not throw");
-    } catch (IllegalArgumentException ex) {
-      // this is what we expect
-    }
+    Assert.assertThrows(
+        "Calling get error on success did not throw",
```

Comment:
When using 'assertXxx' notation, I personally prefer using 'should' in the description like this:

```suggestion
        "Calling .getError() on success should throw",
```

I think this makes reading the test code easier, and still gives a useful hint when the test fails.